index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
+  <script src="js/index.js"></script>
+  <div class="head">
+    <h1>FreecodeCamp Survey Form</h1>
+    <p>Thank you for taking the time to help us improve the platform</p>
+  </div>
+
+  <div class="two">
+    <form>
+      <label for="name">Name</label>
+      <br>
+      <input type="text">
+      <br>
+      <label for="age">Age(optional)</label>
+      <br>
+      <input type="text" name="age">
+      <br>
+      <label for="email">Email</label>
+      <br>
+      <input type="email" name="email">
+      <br>
+      <p>Would you recommend freeCodeCamp to a friend?</p>
+      <br>
+      <input type="radio" name="first" id="">
+      <label for="first">Definately</label>
+      <br>
+      <input type="radio" name="second" id="">
+      <label for="second">Maybe</label>
+      <br>
+      <br>
+      <input type="checkbox" name="third" id="">
+      <label for="third">Front-end Projects</label>
+
+      <br>
+      <input type="checkbox" name="fourth" id="">
+      <label for="fourth">Back-end Projects</label>
+      <br>
+      <br>
+      <input type="submit" value="submit">
+
+    </form>
+  </div>
```

Comment:
Please instead of div enclose this section with a section tag

Reply:
@tingamapuro04, Great, it seems you have an eye on details, as mentioned earlier it's always good to point out why the developer should do the change requested, this not only improves their code but also grow their skills.